dataGenerator.js

```diff
@@ -0,0 +1,94 @@
+const Faker = require('faker');
+
+const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
+
+// Creates an array of businesses
+const createBusinesses = (quantity = 100) => {
+  const businessName = () => `${Faker.random.word()} ${Faker.random.word()}`;
+  const businessArray = [];
+  for (let i = 0; i < quantity; i += 1) {
```

Comment:
Could also start the loop at i = 1 and do <= quantity. Just a style thing, nothing major.